### DISCUSSION TITLE
IDE Improvement: Display Useful Imformation and Operations on IDE

### DISCUSSION THREAD

**Discussion by clean99** [autometrics-dev/autometrics-shared#70] [Ideas]
Instead of having the user redirect to the metrics, can we just display the information inside IDE? like error nums, and average latency. And we can display some useful buttons like "reset" or "refresh" for user.

**Comment by emschwartz** [comment]
We've got a VS Code extension [here](https://github.com/autometrics-dev/vscode-autometrics) and there's an open issue about opening the charts directly in the extension: https://github.com/autometrics-dev/vscode-autometrics/issues/20
